src/main.cpp

```diff
@@ -528,7 +529,8 @@ do_import(std::string file, uint_t limit,
 
             int llen = -1;
             get_line(fin, buff, INPUT_LINE_SIZE, llen);
-            if (llen == -1) {
+            if (llen < 0) {
+                cerr << "No more data to read. Exiting with return value " << llen << std::endl;
```

Comment:
Please remove the print "Exiting with return value", its actually breaking out from loop